src/views/VideoMania/index.jsx

```
@@ -0,0 +1,100 @@
+  handleChangePage,
+  handleChangeRowsPerPage,
+  handleSearch,
+} from "../../utils/common/fetchDataHelpers";
+import { useDispatch, useSelector } from "react-redux";
+import { getVideoCategories, getVideosByCategory } from "../../app/features/videomania";
+import { use } from "react";
+import VideoCard from "../../components/card/VideoCard";
+
+const VideoMania = () => {
+  const [searchQuery, setSearchQuery] = useState("");
+  const [currentPage, setCurrentPage] = useState(1);
```

ESLint

[error] 'currentPage' is assigned a value but never used.

(no-unused-vars)